### PR TITLE
[IBM WebSphere Application Server] Add WebSphere Application Server Integration Package with ThreadPool data stream 

### PR DESCRIPTION
**Pull request by kush-elastic**
- Enhancement

## What does this PR do?

- Generated the skeleton of WebSphere Application Server integration package.
- Added 1 data stream (threadpool).
- Added data collection logic for the data streams.
- Added the ingest pipeline for the data streams.
- Mapped fields according to the ECS schema and added Fields metadata in the appropriate yaml files.
- Added dashboards and visualizations.
- Added pipeline test cases for the data stream.
- Added system test cases for the data stream.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's changelog.yml file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/websphere_application_server directory.
- Run the following command to run tests.
`elastic-package test`

## Related issues

- Relates #2847
- Relates #2922
- Relates #3085
- Relates #3199 

## Screenshots

websphere_application_server-threadpool-dashboard-screenshot

### CODE REVIEW

**Review by ishleenk17 — COMMENTED** [review]
base-fields.yml should have the event.* fields

File: packages/websphere_application_server/manifest.yml [hunk 1]
```diff
@@ -10,13 +10,16 @@ categories:
...
-    type: image/png
   - src: /img/websphere_application_server-jdbc-dashboard-screenshot.png
     title: Websphere Application Server JDBC Dashboard Screenshot
     size: 600x600
     type: image/png
+  - src: /img/websphere_application_server-servlet-dashboard-screenshot.png
+    title: Websphere Application Server Servlet Dashboard Screenshot
+    type: image/png
+  - src: /img/websphere_application_server-threadpool-dashboard-screenshot.png
+    title: Websphere Application Server Servlet Dashboard Screenshot
```
Comment (yug-elastic) [inline packages/websphere_application_server/manifest.yml:21]: nit:
```suggestion
    title: Websphere Application Server ThreadPool Dashboard Screenshot
```

File: packages/websphere_application_server/manifest.yml [hunk 2]
```diff
@@ -10,13 +10,16 @@ categories:
...
+    type: image/png
 icons:
   - src: /img/websphere_application_server-logo.svg
     title: Sample logo
```
Comment (yug-elastic) [inline packages/websphere_application_server/manifest.yml:25]: nit:
```suggestion
    title: IBM WebSphere Application Server logo
```

**Review by yug-elastic — APPROVED** [review]
Please adjust a couple of suggested nits, LGTM otherwise!

**Review by ishleenk17 — APPROVED** [review]

**Pull request merged**